tests/babel-config-invalid/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"jest": {
 		"transform": {
-			".(ts|tsx)": "../../preprocessor.js"
+			".(ts|tsx)": "ts-jest"
```

Comment:
*This applies to all test files with similar changes*

This doesn't look right. Using `../../preprocessor.js` makes sure that the tests run on the current source code. Wouldn't using `ts-jest` (directly or with `<rootDir>` cause the tests to use the previously published version of ts-jest?

Reply:
> Using ../../preprocessor.js makes sure that the tests run on the current source code. Wouldn't using ts-jest (directly or with <rootDir> cause the tests to use the previously published version of ts-jest?

No, I changed `scripts/tests.js` to copy the current source code under each test project's `node_packages/ts-jest/**`:

https://github.com/kulshekhar/ts-jest/pull/373/files#diff-0c26479cc3f82df625bd35543ed2aee0R51